src/sass/variables.sass

```diff
@@ -0,0 +1,7 @@
+$menuHeight: 56vh
+
+$appContainerMarginTop: 2vh
+
+$appContainerPadding: 2vh
+
+$contentAvailableHeight: 100vh
```

Comment:
Ca c'est faux. Justement le contenu n'a pas 100% du viewport disponible pour s'afficher. Quand je parle de content, je parle de ce qu'on va effectivement pouvoir sous la navbar, après avoir retiré la marge avec la navbar et le padding par défaut de l'app pour que l'app respire (et ne touche pas les bords).
Quand tu dis 100vh tu dis déjà "100% de la hauteur du viewport", donc tout le screen visible. Cette déclaration est uniquement destinée à être mise sur `.App` pour dire "mon app n'a jamais le droit de dépasser 100% du viewport". C'est cette déclaration, couplée à celle ci :
```sass
 .app-container
          margin: 10px
          margin-top: 20px
          height: calc(100% - (52px + 20 + 10)px)
```
qui va nous permettre ensuite de faire des sous parties d'app scrollable indépendamment. Est-ce que tu comprends le délire ? Par exemple dans la partie notes pour avoir chaque liste scrollable indépendamment des autres contenus.

Reply:
In fine on aurait plutôt :
```sass
// In variables.sass
$contentMargins: 10px
$contentMarginTop: 20px
$contentAvailableHeight: calc(100vh - $containerMargins - $containerMarginTop)

...

// Dans index.sass
.app
  height: 100vh 
  // On pourrait mettre ça dans une variable $sreenHeight si on voulait rester dans cette logique sémantique des variables
  // et l'utiliser dans la définition de $contentAvailableHeight

  .app-container
    margin: $contentMargins
    margin-top: $contentMarginTop
    height: $contentAvailableHeight

...

// Dans calendar.sass
.calendar
  height: 100% 
  // C'est tout, car du coup le calendrier va avoir une hauteur à 100% celle de son parent, 
  // et donc aura sa height à $contentAvailableHeight, c'est ce qu'on veut. 
  // Donc ton calendrier prendra visuellement tout l'espace "autorisé" par ton layout. Capish ?
```